lib/rory/application.rb

```diff
@@ -39,7 +39,7 @@ def instance
       end
 
       def root=(root_path)
-        @root = Pathname.new(root_path).expand_path
+        $:.unshift @root = Pathname.new(root_path).realpath
```

Comment:
I find require_relative to be brittle when moving files around or extracting a new class into a new file. It is not as easy to understand as doing a require from the root of the app. To address gem file conflicts, if a gem is properly name spacing their gem ie. `require("activesupport/core_ext")` and in my own gem if I wanted to monkey patch that it would look like `require("rory/activesupport/core_ext")`. When good conventions are followed the issues would very rare. I've found `require_relative` to a quick and dirty way to require something when I was too lazy to set up the load path.